### ISSUE TITLE
Request training set data

### ISSUE THREAD

**Issue by hubiyun** [jwohlwend/boltz#124]
I noticed that authors had released pre-processed pdb files, about 60G. Are all of these files used to train the model? Because I noticed that there are structures released after 2021 that also exist in this file. Request pdb data that the author used for training. Or could you please tell me where can I download them?

**Comment by gcorso** [comment]
Hi @Jennyhu666 the raw data was downloaded directly from the PDB website, you can select data used for training taking the following filters:
- Number of chains between 1 and 300
- Release date before 2021-09-30
- Resolution below 9A

**Comment by Jennyhu666** [comment]
Thank you. I want to test the model in practice, and if my target has not released the pdb structure before 2021 then it is not in the model training set. I want to know if there is, for example, a txt file that identifies all the pdb structures used in the training set.

**Comment by gcorso** [comment]
No, unfortunately, we don't have such a list but it shouldn't be too hard to obtain using the PDB API with the filters above
